beets/util/__init__.py

```diff
@@ -949,17 +949,21 @@ def interactive_open(targets, command):
     """
     assert command
 
-    # Split the command string into its arguments.
-    try:
-        args = shlex.split(command)
-    except ValueError:  # Malformed shell tokens.
+    if os.name == 'nt':
         args = [command]
```

Comment:
Does this work for cases when the command has arguments? Like `vim -b`, for example?